docs/containers-storage.conf.5.md

```diff
@@ -167,6 +170,44 @@ Marks devicemapper block device for deferred removal.  If the device is in use w
 
 Specifies the maximum number of retries XFS should attempt to complete IO when ENOSPC (no space) error is returned by underlying storage device. (default: 0, which means to try continuously.)
 
+### STORAGE OPTIONS FOR OVERLAY TABLE
+
+The `storage.options.overlay` table supports the following options:
+
+**ignore_chown_errors** = "true|False"
+  ignore_chown_errors can be set to allow a non privileged user running with a  single UID within a user namespace to run containers. The user can pull and use any image even those with multiple uids.  Note multiple UIDs will be squashed down to the default uid in the container.  These images will have no separation between the users in the container. Only supported for the overlay and vfs drivers.
```

Comment:
The last sentence suggests that we also need a doc entry and checks for `vfs`.

Reply:
Added